BUILD

```diff
@@ -99,7 +120,8 @@ ios_unit_test_suite(
     minimum_os_version = "9.0",
     timeout = "short",
     runners = [
-        ":IPAD_PRO_12_9_IN_9_3",
+# TODO(https://github.com/material-foundation/bazel-workspace-apple/issues/3): Conditionally enable iOS 9 tests on pre-Catalina devices.
```

Comment:
Can this be reverted and still upgrade to 2.0.0? Why does 2.0.0 require changes specifically for Catalina that earlier versions didn't require?